Dockerfile

```diff
@@ -12,19 +12,18 @@ USER nhp
 RUN for DIR in data queue results; do mkdir -p $DIR; done
 
 # Copy dependency files first (optimal caching)
-COPY pyproject.toml uv.lock ./
+COPY --chown=nhp:nhp pyproject.toml uv.lock ./
 
 # Install dependencies only (skip local package)
 RUN uv sync --frozen --no-dev --no-install-project
 
 # Ensure Python can find installed packages and local model
 ENV PATH="/app/.venv/bin:$PATH"
+ENV PYTHONPATH="/app:$PYTHONPATH" 
```

Comment:
I'm pretty sure this is unecessary, and is going to cause the warning from docker seen below about `usage of undefinded variable '$PYTHONPATH'`.

changing `$PATH` above already sets python to default to `/app/.venv/bin/python`, which will see the venv's library

```suggestion
```